### PR TITLE
Added example usage of FFTReal.

### PR DESCRIPTION
**Pull request by ndahlquist**
See https://godoc.org/github.com/ndahlquist/go-dsp/fft#example-FFTReal

### CODE REVIEW

File: fft/fft_test.go [hunk 1]
```diff
@@ -276,3 +278,43 @@ func BenchmarkFFT(b *testing.B) {
...
+
+	// Discretize our function by sampling at 8 points.
+	a := make([]float64, numSamples)
+	for i := 0; i < numSamples; i++ {
+		a[i] = x(i)
+	}
+
+	X := FFTReal(a)
+
+	// Print the magnitude and phase at each frequencey.
```
Comment (mjibson) [inline fft/fft_test.go]: frequencey -> frequency

**Comment by ndahlquist** [comment]
@mjibson, thanks, updated!

**Comment by mjibson** [comment]
Squash these commits into 1 and force push, and I'll merge it.

**Comment by ndahlquist** [comment]
Done

**Pull request merged**